### ISSUE TITLE
box-sizing in ngx-gallery.component.scss

### ISSUE THREAD

**Issue by hijamoya** [kolkov/ngx-gallery#14]
In the original library:
```
    /deep/ {

        * {
            box-sizing: border-box;
        }
  ...
```
In this library:
```
* {
  box-sizing: border-box;
}
```

This make global box-sizing change after open the gallery which causes some css issues.

**Comment by kolkov** [comment]
Yes, at the new version of Angular `/deep/` is not working. ( but I tried this at Next versions, may be it working in release,  I don't know.

**Comment by hijamoya** [comment]
I can confirm that ::ng-deep still working.

**Comment by totize** [comment]
Indeed, ::ng-deep is still working.

Could you maybe update your code? That would fix many webpages' style issues I bet :)

**Comment by PedroJesusRomeroOrtega** [comment]
While updating to angular 9, in the [official update guide](https://update.angular.io/#7.2:9.0l3) says this:
* Replace /deep/ with ::ng-deep in your styles, [read more about angular component styles and ::ng-deep](https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep). /deep/ and ::ng-deep both are deprecated but using ::ng-deep is preferred until the shadow-piercing descendant combinator is [removed from browsers and tools](https://www.chromestatus.com/features/6750456638341120) completely.

In the meantime, I've fixed by adding
```
ngx-gallery ::ng-deep .ngx-gallery-thumbnails .ngx-gallery-thumbnail {
  border: none !important;
}
```
in the component.scss file.
I think it isn't the best approach because I don't like _!important_, but I haven't found another solution.

**Comment by kolkov** [comment]
@PedroJesusRomeroOrtega thanks, are there any other ways in Angular repo issues?

**Comment by PedroJesusRomeroOrtega** [comment]
Sorry, I don't know other ways

**Comment by german-e-mas** [comment]
Nice catch. Thanks @hijamoya , I was wondering why my styles suddenly broke when updating to this library.

As for the styles, I've been using @PedroJesusRomeroOrtega 's solution. `::ng-deep` works (i haven't find a better solution that doesn't use `!important` though). It would be nice to have a better way to control the border style of thumbnails.

**Comment by corecodingltd** [comment]
This is still an issue for me, it really messes up the CSS of the rest of the site. Can you not structure your CSS better and not use global declarations?
